deployment/huggingface/yolos-seq2seq.ipynb

```diff
@@ -0,0 +1,1699 @@
+      "source": [
+        "urls = urls[:10]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tVb8ea1GjPbK"
+      },
+      "outputs": [],
+      "source": [
+        "def process_image(url):\n",
+        "    return deployed_model.predict(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "jbSdW0lUjPmG"
+      },
+      "outputs": [],
+      "source": [
+        "results = []\n",
+        "start_time = time.time()\n",
+        "\n",
+        "with concurrent.futures.ThreadPoolExecutor() as executor:\n",
+        "    for url, result in zip(urls, executor.map(process_image, urls)):\n",
+        "        results.append([url, result])\n",
+        "        \n",
+        "end_time = time.time()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "JatFuc7-jPqz"
+      },
+      "outputs": [],
+      "source": [
+        "total_time = end_time - start_time\n",
+        "total_time = time.strftime('%Hh %Mm %Ss', time.gmtime(total_time))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "lIjZzHLojj61",
+        "outputId": "1e4d2e9a-15c1-49dd-ef3e-61f6501f1720"
+      },
+      "outputs": [],
+      "source": [
+        "print(f\"Processing Time: {total_time} for {len(urls)} images.\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VGAHxQWWxuUZ"
+      },
+      "outputs": [],
+      "source": [
+        "results_rows = []\n",
+        "\n",
+        "for img_result in results:\n",
+        "    url = img_result[0]\n",
+        "    entities = img_result[1]\n",
+        "\n",
+        "    for item in entities:\n",
+        "        item = item.copy()\n",
+        "        item.update(item.pop('box'))\n",
+        "        item['url'] = url\n",
+        "        results_rows.append(pd.Series(item))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "a3oMDUw0jskb"
+      },
+      "outputs": [],
+      "source": [
+        "df = pd.concat(results_rows, axis=1).T"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 655
+        },
+        "id": "Kj0VktHojsp0",
+        "outputId": "bbced217-26d3-4fa8-fc4f-00c698cb989b"
+      },
+      "outputs": [],
+      "source": [
+        "df"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "l7tXPk3rjsv1"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint.delete()"
```

Comment:
Why delete the endpoint here?  The model would still be registered, and a user may want to experiment further.  As a side note though, I do think it would be cool if we had an easy way to tear down whatever our notebooks create, like an optional cell at the end that could be run to clean up.

Reply:
It serves as a reminder for the user to delete the endpoint. Without it, users have been leaving their endpoints running because deleting it never occurs to them, which takes up the endpoint name used in the notebook and eats up k8s resources.

> a user may want to experiment further.

I think if a user doesn't want to delete the endpoint, they can choose to not run the cell.